README.md

```
@@ -25,32 +26,61 @@ A modern, interactive map dashboard built with React and Vite. This project prov
+pnpm run preview
+```
+
+Run linter:
+
+```sh
+pnpm run lint
+```
+
 ## Project Structure
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)